webapp/login/views.py

```diff
@@ -77,8 +77,6 @@ def after_login(resp):
         return flask.redirect(LOGIN_URL)
 
     account = publisher_api.get_account(flask.session)
-    owned, shared = logic.get_snap_names_by_ownership(account)
-    flask.session["user_shared_snaps"] = shared
```

Comment:
This was adding an array of snap-names to the session e.g. ["snap1", "snap2", "snap3"] etc. If someone is a collaborator on many snaps, this list could be enormous.